apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.ts

```diff
@@ -84,9 +84,10 @@ export class NewInventoryAdvanceSearch implements OnInit {
             this.isMobileScreen = result.matches;
         });
         this.reportType = this.inputData?.reportType;
-        if (this.inputData?.stockReportRequest) {
+        if (this.inputData?.stockReportRequest || this.inputData?.stockReportRequestExport) {
```

Comment:
check if it is same interface then use  this.inputData?.stockReportRequest

Reply:
done